lib/abide_dev_utils/xccdf.rb

```diff
@@ -236,7 +236,6 @@ def gen_map(dir: nil, type: 'CIS', parent_key_prefix: '', **_)
         else
           mapping_dir = ''
         end
-        parent_key_prefix = parent_key_prefix.nil? ? nil : ''
```

Comment:
This was in there because if you explicitly pass a value of `nil` into that parameter, it will cause exceptions to be thrown later in the code.

Reply:
Fair enough